styles/color-mixins.less

```diff
@@ -80,7 +80,7 @@
 	opacity: 1;
 	.sand-bg-colors({
 		background-color: @sand-spotlight-bg-color;
-		filter+: drop-shadow(@sand-spotlight-focus-shadow);
+		box-shadow: @sand-spotlight-focus-shadow;
```

Comment:
I think we need to add `filter` to https://github.com/enactjs/sandstone/pull/956/files#diff-6fa658458e34133889fd42ba25feceb3b5bdcc8312532446734665242427273cR159.
And change https://github.com/enactjs/sandstone/pull/956/files#diff-6fa658458e34133889fd42ba25feceb3b5bdcc8312532446734665242427273cR71 to `box-shadow`.